cmd/execute.go

```diff
@@ -104,8 +108,10 @@ func executeTransformers() {
 		go watchEthEvents(&ew, &wg)
 	}
 
+	storageHealthCheckMessage := []byte("storage watcher starting\n")
```

Comment:
Somewhat minor - but for consistency it may be worthwhile to initialize these two variables inside the if check like they are above. I got a bit confused with statusWriter, since it's initialized twice in two different scopes.